--- app/src/test/kotlin/com/bitperfect/app/ui/ComponentsTest.kt
+++ app/src/test/kotlin/com/bitperfect/app/ui/ComponentsTest.kt
@@ -26,6 +26,7 @@
         Mockito.`when`(mockViewModel.driveStatus).thenReturn(MutableStateFlow(DriveStatus.Connecting(null)))
         Mockito.`when`(mockViewModel.discMetadata).thenReturn(MutableStateFlow(null))
         Mockito.`when`(mockViewModel.coverArtUrl).thenReturn(MutableStateFlow(null))
+        Mockito.`when`(mockViewModel.isKeyDisc).thenReturn(MutableStateFlow(false))

         composeTestRule.setContent {
             DeviceList(viewModel = mockViewModel)
@@ -42,6 +43,7 @@
         Mockito.`when`(mockViewModel.driveStatus).thenReturn(MutableStateFlow(DriveStatus.Empty(DriveInfo("ASUS", "BW-16D1HT", true, 0, 0, ""))))
         Mockito.`when`(mockViewModel.discMetadata).thenReturn(MutableStateFlow(null))
         Mockito.`when`(mockViewModel.coverArtUrl).thenReturn(MutableStateFlow(null))
+        Mockito.`when`(mockViewModel.isKeyDisc).thenReturn(MutableStateFlow(false))

         composeTestRule.setContent {
             DeviceList(viewModel = mockViewModel)
@@ -58,6 +60,7 @@
         Mockito.`when`(mockViewModel.driveStatus).thenReturn(MutableStateFlow(DriveStatus.DiscReady(DriveInfo("ASUS", "BW-16D1HT", true, 0, 0, ""), null, null)))
         Mockito.`when`(mockViewModel.discMetadata).thenReturn(MutableStateFlow(null))
         Mockito.`when`(mockViewModel.coverArtUrl).thenReturn(MutableStateFlow(null))
+        Mockito.`when`(mockViewModel.isKeyDisc).thenReturn(MutableStateFlow(false))

         composeTestRule.setContent {
             DeviceList(viewModel = mockViewModel)
